### DISCUSSION TITLE
关于地域限制问题是政策问题还是bug?

### DISCUSSION THREAD

**Discussion by QAQQL** [community/community#51757] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/51136763/229361582-7bd594ec-dcde-4db9-a530-c6a2f6a7b89f.png)
作为一个软件服务商,您的地域政策问题这边暂时不做评价,您不提供服务是否应该进行退款或提前声明?????

**Comment by Akash1134** [comment]
Hi @QAQQL ,

Thanks for getting in touch.

Due to current trade controls, your GitHub account has been restricted from making use of Copilot. This is a region-based restriction and applies to all users located in some trade-restricted countries, being flagged for the same earlier, and is separate from other GitHub services. You can read more about it here:

[GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls)

Sorry if that wasn't what you were hoping to hear. Thanks!

**Comment by QAQQL** [comment]
我浏览了关于限制问题的描述 限制地区包括以下： 克里米亚、顿涅茨克和卢甘斯克的分离主义地区、古巴、伊朗、朝鲜和叙利亚。 可我是中国用户，并未在上述地区使用，请告知我具体的账号限制原因和解封策略，如果不能解封，请给我提供退款途径，因为在种种情况下，你们已经无法为我的订阅提供服务

**Reply by ktKongTong** [reply]
同样的问题，一种可能的解决方案如下：
在账单信息页面完善你的支付信息，https://github.com/settings/billing/payment_information ，完善你的支付信息。

我在未完善的情况下，https://github.com/settings/billing/spending_limit ，该页面会提示我如下信息。
![image](https://user-images.githubusercontent.com/44502608/229827660-8da1b5d3-4f34-430a-8302-37e5c07b070d.png)

当我完善之后，该警告信息消失，copilot 也变得可用。


但是判断信息是否完善的机制似乎很微妙。我之前是有相关信息的，但是姓名部分均只取了我 id 两个字的首字母（kt），在我填写完整拼音之后kongtong，警告信息便消失，copilot 也变得可用。于是我反复修改常识测试警告触发的机制，结果似乎处罚风控，暂时不可以修改支付信息，copilot也用不了了 :-(。

**Reply by QAQQL** [reply]
![图片](https://user-images.githubusercontent.com/51136763/229968544-3eda7540-00ce-4c0b-b6e3-bd3399121de2.png)
我没有这个提示,甚至我都不知道我的copilot是怎么停止我的服务的

**Reply by QAQQL** [reply]
![图片](https://user-images.githubusercontent.com/51136763/229968717-34f9d104-db5d-4e66-97cb-1b3e06338752.png)
我看到在另一个页面有类似提示
https://github.com/settings/billing/spending_limit
请待我完善后再反馈

**Reply by ktKongTong** [reply, marked as answer]
> ![图片](https://user-images.githubusercontent.com/51136763/229968717-34f9d104-db5d-4e66-97cb-1b3e06338752.png) 我看到在另一个页面有类似提示 https://github.com/settings/billing/spending_limit 请待我完善后再反馈

是同样的提示。

> ![图片](https://user-images.githubusercontent.com/51136763/229968544-3eda7540-00ce-4c0b-b6e3-bd3399121de2.png) 我没有这个提示,甚至我都不知道我的copilot是怎么停止我的服务的

看起来是同样的问题，可能也是因为你姓名字段太短了，刚才我修改 billing information 之后，再添加了一个 payment method 就可以了

**Reply by QAQQL** [reply]
可以确认的是,这不是地域限制,而仅仅是名称限制规则改了触发的限制
不过这个提示过于迷惑,以至于一度认为是location的问题

**Reply by QAQQL** [reply]
修改之后copilot恢复了正常,感谢您的解答

**Reply by baiwandanjiwang** [reply]
![屏幕截图 2024-12-18 145005](https://github.com/user-attachments/assets/68478fec-6823-4b96-b04d-e4e580eb3b44)
那我这样无法更改账单信息怎么办

**Reply by baiwandanjiwang** [reply]
github copilot也用不了

**Reply by SheepTAO** [reply]
> ![屏幕截图 2024-12-18 145005](https://private-user-images.githubusercontent.com/119772071/396792031-68478fec-6823-4b96-b04d-e4e580eb3b44.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgwMDIwNjAsIm5iZiI6MTc1ODAwMTc2MCwicGF0aCI6Ii8xMTk3NzIwNzEvMzk2NzkyMDMxLTY4NDc4ZmVjLTY4MjMtNGI5Ni1iMDRkLWU0ZTU4MGViM2I0NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkxNlQwNTQ5MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mOTVlYmUxMDI5YWM4NDc4OTJlZmE1MTdlMjc3ZGQwY2UwODRmYWQ0NjE1OGVhNTE4NTU0MGQyYWUzYzk0MDExJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.aquVwTy6sHkyqXz2a6WjL57j5ehqJY3shJaQmf5fdM4) 那我这样无法更改账单信息怎么办

请问你解决了吗，我现在也遇到了一样的问题

**Reply by median-dxz** [reply]
> > ![屏幕截图 2024-12-18 145005](https://private-user-images.githubusercontent.com/119772071/396792031-68478fec-6823-4b96-b04d-e4e580eb3b44.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgwMDIwNjAsIm5iZiI6MTc1ODAwMTc2MCwicGF0aCI6Ii8xMTk3NzIwNzEvMzk2NzkyMDMxLTY4NDc4ZmVjLTY4MjMtNGI5Ni1iMDRkLWU0ZTU4MGViM2I0NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkxNlQwNTQ5MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mOTVlYmUxMDI5YWM4NDc4OTJlZmE1MTdlMjc3ZGQwY2UwODRmYWQ0NjE1OGVhNTE4NTU0MGQyYWUzYzk0MDExJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.aquVwTy6sHkyqXz2a6WjL57j5ehqJY3shJaQmf5fdM4) 那我这样无法更改账单信息怎么办
> 
> 请问你解决了吗，我现在也遇到了一样的问题

我也遇到了一样的问题

**Reply by karlYangYang** [reply]
> github copilot也用不了

我也是同样的问题，无法修改，github发邮件说是由于贸易限制，现在支付信息完全被锁定了
